test/integration/extension/lib/bigcommerce/CartRepositoryTest.js

```diff
@@ -63,8 +76,38 @@ describe('BigCommerceCartRepository - integration', () => {
     return subjectUnderTest.destroy().should.eventually.be.fulfilled
   })
 
-  it('should not give back previously destroyed cart', function () {
+  it('should not give back previously destroyed cart', async function () {
+    storageMock.expects('get').once().returns(cartId)
+    await subjectUnderTest.load().should.eventually.be.fulfilled.and.be.equal(null)
```

Comment:
There is no point in using eventually sintax if there is an await here. Also, still a promise is returned - it still might get rejected. Will it be handled by mocha this way?

Reply:
I just throw an Error inside of load() and with the eventually syntax it will be handled correctly.

Output: "AssertionError: expected promise to be fulfilled but it was rejected with 'Error: test message"

I won't change the code here because it does what we would expect